build.gradle.kts

```diff
@@ -207,7 +206,7 @@ tasks.register("createApp") {
          if (process1.waitFor() != 0) {
             throw GradleException("Error while creating app directory")
          }
-         val plistfilename = "$buildDir/dist/" + appName + ".app/Contents/Info.plist"
+         val plistfilename = appDirname + "/Contents/Info.plist"
```

Comment:
Why concatenation instead of plain interpolation you had elsewhere (-> `"$appDirname/Contents/Info.plist"`)?

Reply:
I had tried "$appDirname" and similar things in the inputs and outputs parts. It didn't work. I had to use the ext.get approach. So I assumed that only worked for system level names such as $buildDir. There is probably something else going on, such as missing type information. It does seem to work for local variables, so I could do that here. But without the ext.get for the local variable, it still fails.

Hmmm. I see that plistfilename has also not been renamed.

Reply:
Done.